File: FrontEnd/src/app/models/questionary.ts
import { Question } from './question';

export class Questionary {
    id?: number;
    name: string;
    description: string;
    create?: Date;
    listQuestions: Question[];

    constructor(name: string, description: string, create: Date, listQuestions: Question[]) {
        this.name = name;
        this.description = description;
        this.create = create;
        this.listQuestions = listQuestions;
    }
}

File: BackEnd/BackEnd/Domain/Models/Questionary.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace BackEnd.Domain.Models
{
    public class Questionary
    {
        public int Id { get; set; }
        [Required]
        [Column(TypeName = "varchar(100)")]
        public string Name { get; set; }
        [Required]
        [Column(TypeName = "varchar(150)")]
        public string Description { get; set; }
        public DateTime Create { get; set; }
        public bool IsActive { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public List<Question> listQuestions { get; set; }
    }
}
